data_pipeline/generic_web_api/generic_web_api_config.py

```diff
@@ -82,6 +82,11 @@ def __init__(
         ).get("configurableParameters", {})
         self.default_start_date = configurable_parameters.get(
             "defaultStartDate", None)
+        default_start_end_date_diff_in_days = (
+            configurable_parameters.get(
+                "defaultDaysDiffFromStartTillEnd", None
```

Comment:
minor: why is it called default? It seems to be the only configuration value we use.

minor: usually we put the type / unit at the end, like in the variable name. I was trying to think what name I personally would think of initially. Maybe something like `queryPeriodInDays`. Although your name refers to the affected parameters and may be better that way.